app/components/provider_interface/application_choice_header_component.rb

```diff
@@ -90,10 +90,9 @@ def provider_cannot_respond?
     end
 
     def show_inset_text?
-      flash.empty? && (
-        respond_to_application? || deferred_offer_wizard_applicable? ||
-        rejection_reason_required? || provider_cannot_respond?
-      ) || waiting_for_interview?
+      respond_to_application? || deferred_offer_wizard_applicable? ||
+        rejection_reason_required? || provider_cannot_respond? ||
+        waiting_for_interview?
```

Comment:
Can we juts remove `show_inset_text?` entirely?
Are there any other possible states?

Reply:
Not really, because it controls whether the `govuk-inset-text` surrounding structure is created. If you remove this check, you'll have extra whitespace and a nice short grey bar where the inset text would appear.

Reply:
So there are other states as well that this doesn't apply to.

Reply:
Correct